trainbot.service

```diff
@@ -0,0 +1,19 @@
+# This unit assumes that trainbot is run as a systemd user service, that the
+# trainbot git repo has been cloned to ~/trainbot, and that the data folder
+# should default to ~/trainbot/data.
+#
+# Correctness can be checked using: systemd-analyze verify ./trainbot.service
+
+[Unit]
+Description=onlytrains trainbot
+Documentation=https://github.com/jo-m/trainbot/
+
+[Service]
+Type=exec
+WorkingDirectory=%h/trainbot
+ExecStart=%h/trainbot/build/trainbot \
```

Comment:
ehh, maybe we should be running `trainbot-arm64` here? Not sure, what paths to expect.

Reply:
Generelly it's really easy here for a newcomer to deviate from the expected paths and then this service file won't work without adaptations.